pkgs/development/python-modules/pycmarkgfm/default.nix

```diff
@@ -0,0 +1,26 @@
+{ lib, buildPythonPackage, fetchPypi, isPy27, cffi, pytest }:
+
+buildPythonPackage rec {
+  pname = "pycmarkgfm";
+  version = "1.0.1";
+  disabled = isPy27;
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "0wkbbma214f927ikn3cijxsrzkmm5cqz1x4fimrwx9s2wfphj250";
+  };
+
+  propagatedBuildInputs = [ cffi ];
+
+  checkInputs = [ pytest ];
+  checkPhase = ''
+    pytest tests
```

Comment:
Can you use pytestCheckHook? For a rather complete example you can look at https://github.com/NixOS/nixpkgs/blob/master/pkgs/servers/home-assistant/default.nix#L161.

Reply:
I tried doing this, sadly this package relies on building a native extension (builds `cmark.so` as part of the setup), and it seems like the pytestCheckHook runs in an environment where that lib is not available. It works fine with the current code. Do you have advice on how to overcome this limitation? Otherwise I'll leave this as-is. Note that [cmarkgfm](https://github.com/NixOS/nixpkgs/blob/master/pkgs/development/python-modules/cmarkgfm/default.nix#L17) has virtually the same build setup, and uses the `checkPhase = pytest` approach.

Reply:
Can you add a comment with that?

Reply:
@SuperSandro2000 done.